db/fixtures/miq_shortcuts.yml

```diff
@@ -274,6 +274,21 @@
   :url: /container_topology
   :rbac_feature_name: container_topology_view
   :startup: true
+- :name: migration
+  :description: Migration Plans
```

Comment:
To be consistent with other entries here, maybe we should match the menu structure and include the top-level "Migration / ..."?

Migration / Migration Plans
Migration / Infrastructure Mappings
Migration / Migration Settings

Reply:
You're right, thanks. Fixed.